tests/garage/.pylintrc

```diff
@@ -0,0 +1,52 @@
+[MASTER]
+# Check docstring completeness
+load-plugins = pylint.extensions.docparams, pylint.extensions.docstyle
+
+# Unit tests have a special configuration which is checked separately
+ignore = tests/garage
```

Comment:
Is this being ignored? Otherwise wouldn't putting this in the tests .pylintrc ignore all tests?

Reply:
oh good point -- in this case it's harmless because tests/garage would have to be a relative path from `pwd` -- so this will ignore anything in `tests/garage/tests/garage/*` -- I will make a follow-up PR to fix this, since the CI takes so long.

Reply:
Sure. I think this actually wouldn't work even if there was something in `tests/garage/tests/garage/`. Pylint's documentation says this about `ignore`
```
--ignore=<file[,file...]>
    Add files or directories to the blacklist. They should be base names, not paths.
```

So even when I do `pylint --ignore=garage/np garage` or `pylint --ignore=src/garage/np garage`, I still see output for that directory. Even `pylint --ignore=garage/np garage/np` doesn't seem ti ignore anything.

Or maybe I don't understand how to use the ignore parameter.